reo/nested_inputs.py

```diff
@@ -160,6 +160,11 @@ def list_of_dict(input):
       "default": 120,
       "description": "The number of seconds allowed before the decomposition subproblem optimization times out"
     },
+    "add_soc_incentive": {
+      "type": "bool",
+      "default": True,
+      "description": "Adds a small incentive for battery state of charge (e.g., to dispatch for resilience) if true, and removes the incentive (e.g., to dispatch for peak-shaving) otherwise"
```

Comment:
@zolanaj I think that the second example of "to dispatch for peak-shaving" is misleading since the objective is always to minimize cost, which includes peak-shaving if there are demand charges.  Perhaps update the description to: "If True then a small incentive to keep the battery state-of-charge high is added to the objective of the optimization." ?

Reply:
Thanks for your feedback on this; the description has been updated in commit 5457c1f9dca8bb1bc66317e683583a723f981755